src/main/proto/tech/figure/servicing/v1beta1/loan_state.proto

```diff
@@ -17,59 +16,66 @@ message LoanStructureState {
   repeated LoanState loan_states    = 2; // Loan state data for loans included in the structure
 }
 
+/*
+List of loan states for a single loan
+ */
+message LoanStates {
+  // Static Fields - repeated so that owner can
+  tech.figure.util.v1beta1.UUID      loan_id                = 1 [(validate.rules).message.required = true]; // Loan Identifier
+  tech.figure.util.v1beta1.AssetType asset_type             = 2 [(validate.rules).message.required = true]; // Asset type (See docs/util.md)
+  tech.figure.util.v1beta1.Borrowers current_borrower_info  = 3;                                            // Borrower(s), co-signers, etc
+  repeated LoanState                 loan_state             = 10;                                           // Individual loan states appended throughout life of loan
```

Comment:
Is there any reason why we're not using the plural name since this is a repeated field?